src/content/en/fundamentals/getting-started/service-worker/learn-more-get-help.markdown

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Merging it with previous broadens the scope of the previous. That would require new intro and a second level of internal TOC. Since they don't logically fit together I don't think the word is justified.